### PR TITLE
Link To deletion of UserTag on profile page broken . #1860

### PR DESCRIPTION
**Pull request by sagarpreet-chadha**
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

@jywarren sir...kindly review this PR .

* [X] all tests pass -- `rake test:all`
* [X] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request is descriptively named with #number reference back to original issue

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

File: test/functional/user_tags_controller_test.rb [hunk 1]
```diff
@@ -102,7 +102,7 @@ def setup
       @controller = old_controller
 
       UserSession.create(users(:bob))
-      post :delete, id: 9999
+      post :delete, id: 9999 , name: user_tag.value
```
Comment (jywarren) [inline test/functional/user_tags_controller_test.rb]: Does this have to be method: delete, somehow?

**Comment by sagarpreet-chadha** [comment]
Problem was in user_tag_controller_test.rb , id of TAG was passed in many tests but we have to pass id of USER instead .

**Comment by jywarren** [comment]
Ah, super, great detective work! Is this ready to merge then? Thank you so
much!!!

**Comment by sagarpreet-chadha** [comment]
Yes sir...completely resolves the issue now  ! Thank you !

**Comment by jywarren** [comment]
🎉 awesome!!!

**Pull request merged**